Automattic/wp-calypso
# Global nav i4 - add selected global tab styles

Related to # Automattic/wp-calypso#93038 Automattic/wp-calypso#93037  Automattic/wp-calypso#93039

## Proposed Changes

Applies the styles outlined in the issues above.
* Special selected styles for sites dashboard, me, and reader tabs in global masterbar.
* Updates global masterbar text/icon colors and submenu background colors as outlined in Automattic/wp-calypso#93039
* Adds a small amount of top padding for the content frame back to mobile, to support the tab styles as recommended in Automattic/wp-calypso#93038

BEFORE
![new-global-masterbar-styles-before](https://github.com/user-attachments/assets/4492945d-90d3-425f-b088-bc1420d29cbb)


AFTER
![new-global-masterbar-after](https://github.com/user-attachments/assets/137dcf10-a426-4b3f-a0e0-e4b8dda5a60d)


## Why are these changes being made?


* Per design request, iteration to make new interfaces look better in global area, and pe7F0s-1XD-p2

## Testing Instructions



* Run this branch of calypso.
* Smoke test the new styles in the global masterbar.
* Verify there are no regressions on the site level masterbar like my home.
* Test mobile/tablet viewports, and with various browsers.

## Pre-merge Checklist
